gatsby_fp/src/styles/site.css

```diff
@@ -1,15934 +1,5 @@
-  font-style: normal;
-  font-weight: 700;
-  font-size: 14px;
-  font-size: 0.875rem;
-  text-transform: uppercase;
-  display: inline-block;
-  position: relative;
-  color: #111111;
-  letter-spacing: 0.6px;
-  margin-right: 20px;
-  padding-right: 25px;
-  color: #fff;
-}
-.homepage_callout_spotlight_link:last-child {
-  margin-right: 0;
-}
-.homepage_callout_spotlight_link:after {
-  position: absolute;
-  display: inline-block;
-  content: "\2192";
-  font-family: sans-serif;
-  margin-left: 5px;
-  -webkit-transition: -webkit-transform 0.25s linear;
-  transition: -webkit-transform 0.25s linear;
-  transition: transform 0.25s linear;
-  transition: transform 0.25s linear, -webkit-transform 0.25s linear;
-}
-.no-touchevents .homepage_callout_spotlight_link:hover {
-  color: #9d1535;
-}
-.no-touchevents .homepage_callout_spotlight_link:hover:after {
-  -webkit-transform: translateX(3px);
-  -ms-transform: translateX(3px);
-  transform: translateX(3px);
-}
-@media only screen and (max-width: 499px) {
-  .homepage_callout_spotlight_link {
-    width: 100%;
-    margin: 8px 0;
-  }
-}
-.no-touchevents .homepage_callout_spotlight_link:hover {
-  color: rgba(255, 255, 255, 0.5);
-}
-@media only screen and (max-width: 739px) {
-  .homepage_callouts_people .fs-carousel-enabled .fs-carousel-controls {
-    display: none;
-  }
-}
-.homepage_callouts_people
-  .fs-carousel-enabled
-  .fs-carousel-pagination
-  .fs-carousel-page {
-  background: #666666;
-}
-.homepage_callouts_people
-  .fs-carousel-enabled
-  .fs-carousel-pagination
-  .fs-carousel-page.fs-carousel-active {
-  background: #9d1535;
-}
-@media only screen and (min-width: 740px) {
-  .homepage_callouts_people .fs-carousel-enabled .fs-carousel-pagination {
-    display: none;
-  }
-}
-@media only screen and (min-width: 980px) {
-  .homepage_callouts_people_inner {
-    margin-left: -40px;
-  }
-}
-@media only screen and (min-width: 1220px) {
-  .homepage_callouts_people_inner {
-    margin-left: -50px;
-  }
-}
-@media only screen and (max-width: 499px) {
-  .homepage_callouts_people_carousel {
-    width: auto !important;
-    margin-right: -20px;
-    margin-left: -20px;
-  }
-}
-.homepage_callout_people {
-  display: block;
-}
-.homepage_callout_people_figure {
-  position: relative;
-}
-.homepage_callout_people_content {
-  position: absolute;
-  width: 100%;
-  -webkit-font-smoothing: auto;
-  -moz-osx-font-smoothing: grayscale;
-  -webkit-transform: translateZ(0);
-  transform: translateZ(0);
-  bottom: 0;
-  left: 0;
-  background: rgba(17, 17, 17, 0.7);
-  padding: 30px;
-}
-@media only screen and (min-width: 740px) {
-  .homepage_callout_people_content {
-    padding-right: 75px;
-    padding-left: 75px;
-  }
-}
-@media only screen and (min-width: 980px) {
-  .homepage_callout_people_content {
-    min-height: 100px;
-  }
-}
-.homepage_callout_people_heading {
-  font-family: "Trade Gothic W01", Arial, Helvetica, sans-serif
```

Comment:
This is the minified version. Try using the "no-min" (non-minified) file from https://bitbucket.org/stevens_edu/stevens_drupal/src/master/docroot/sites/all/themes/stevens/css/